guacamole-docker/bin/build-guacamole.sh

```diff
@@ -43,10 +43,14 @@
 ##     to empty string. Can be set to "lgpl-extensions" to e.g. include
 ##     RADIUS authentication extension.
 ##
+## @param MAVEN_ARGUMENTS
+##     Any number of arbitrary extra arguments to be passed to maven.
+##
 
 BUILD_DIR="$1"
 DESTINATION="$2"
 BUILD_PROFILE="$3"
+MAVEN_ARGUMENTS="$4"
```

Comment:
I'm not sure if there is a better way to do this ... as it is you'd have to invoke it like
`build-guacamole.sh "/tmp/some-path" "/opt/guacamole" "" -DskipTests=false`  - inserting a dummy argument to pad out the argument count to make sure the maven arguments get to the right place. 

Thoughts?

Reply:
`build-guacamole.sh` could leverage environment variables directly, using positional arguments only for `BUILD_DIR` and `DESTINATION`. That would leave us with `BUILD_DIR` and `MAVEN_ARGUMENTS` as build arguments that could be defined in the `Dockerfile`.

Reply:
Seems reasonable to me.